acceptance-tests/tests/common.go

```diff
@@ -0,0 +1,355 @@
+	RegisterTestingT                      = gomega.RegisterTestingT
+	InterceptGomegaFailures               = gomega.InterceptGomegaFailures
+	Ω                                     = gomega.Ω
+	Expect                                = gomega.Expect
+	ExpectWithOffset                      = gomega.ExpectWithOffset
+	Eventually                            = gomega.Eventually
+	EventuallyWithOffset                  = gomega.EventuallyWithOffset
+	Consistently                          = gomega.Consistently
+	ConsistentlyWithOffset                = gomega.ConsistentlyWithOffset
+	SetDefaultEventuallyTimeout           = gomega.SetDefaultEventuallyTimeout
+	SetDefaultEventuallyPollingInterval   = gomega.SetDefaultEventuallyPollingInterval
+	SetDefaultConsistentlyDuration        = gomega.SetDefaultConsistentlyDuration
+	SetDefaultConsistentlyPollingInterval = gomega.SetDefaultConsistentlyPollingInterval
+	NewWithT                              = gomega.NewWithT
+	NewGomegaWithT                        = gomega.NewGomegaWithT
+	Default                               = gomega.Default
+	Equal                = gomega.Equal
+	BeEquivalentTo       = gomega.BeEquivalentTo
+	BeIdenticalTo        = gomega.BeIdenticalTo
+	BeNil                = gomega.BeNil
+	BeTrue               = gomega.BeTrue
+	BeFalse              = gomega.BeFalse
+	HaveOccurred         = gomega.HaveOccurred
+	Succeed              = gomega.Succeed
+	MatchError           = gomega.MatchError
+	BeClosed             = gomega.BeClosed
+	Receive              = gomega.Receive
+	BeSent               = gomega.BeSent
+	MatchRegexp          = gomega.MatchRegexp
+	ContainSubstring     = gomega.ContainSubstring
+	HavePrefix           = gomega.HavePrefix
+	HaveSuffix           = gomega.HaveSuffix
+	MatchJSON            = gomega.MatchJSON
+	MatchXML             = gomega.MatchXML
+	MatchYAML            = gomega.MatchYAML
+	BeEmpty              = gomega.BeEmpty
+	HaveLen              = gomega.HaveLen
+	HaveCap              = gomega.HaveCap
+	BeZero               = gomega.BeZero
+	ContainElement       = gomega.ContainElement
+	BeElementOf          = gomega.BeElementOf
+	ConsistOf            = gomega.ConsistOf
+	ContainElements      = gomega.ContainElements
+	HaveKey              = gomega.HaveKey
+	HaveKeyWithValue     = gomega.HaveKeyWithValue
+	BeNumerically        = gomega.BeNumerically
+	BeTemporally         = gomega.BeTemporally
+	BeAssignableToTypeOf = gomega.BeAssignableToTypeOf
+	Panic                = gomega.Panic
+	PanicWith            = gomega.PanicWith
+	BeAnExistingFile     = gomega.BeAnExistingFile
+	BeARegularFile       = gomega.BeARegularFile
+	BeADirectory         = gomega.BeADirectory
+	HaveHTTPStatus       = gomega.HaveHTTPStatus
+	And                  = gomega.And
+	SatisfyAll           = gomega.SatisfyAll
+	Or                   = gomega.Or
+	SatisfyAny           = gomega.SatisfyAny
+	Not                  = gomega.Not
+	WithTransform        = gomega.WithTransform
+)
```

Comment:
Where exactly these variables used..? I can't see any usage at least here

Reply:
ginkgo needs these vars its auto generated by ginkgo / if we dont want this we have to use ginkgo as `.` import which will cause linter to fail